commit 4f7f31d5808fef2ec70cd1117a0bb1662378dbea
Author: Mad Jami <xmadjamix@gmail.com>
Date:   Wed Apr 14 00:34:04 2021 -0400

    Remove EUICC and stop SIM MANAGER Craching error

diff --git a/bonito/Android.bp b/bonito/Android.bp
index d2e13136..916b768d 100644
--- a/bonito/Android.bp
+++ b/bonito/Android.bp
@@ -231,30 +231,6 @@ android_app_import {
 	product_specific: true,
 }
 
-android_app_import {
-	name: "EuiccGoogle",
-	owner: "google",
-	apk: "proprietary/product/priv-app/EuiccGoogle/EuiccGoogle.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "EuiccSupportPixel",
-	owner: "google",
-	apk: "proprietary/product/priv-app/EuiccSupportPixel/EuiccSupportPixel.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
 android_app_import {
 	name: "HotwordEnrollmentOKGoogleRT5514",
 	owner: "google",
diff --git a/bonito/bonito-vendor.mk b/bonito/bonito-vendor.mk
index da60194e..1192926a 100644
--- a/bonito/bonito-vendor.mk
+++ b/bonito/bonito-vendor.mk
@@ -139,8 +139,6 @@ PRODUCT_COPY_FILES += \
     vendor/google/bonito/proprietary/product/lib64/libsecureuisvc_jni.so:$(TARGET_COPY_OUT_PRODUCT)/lib64/libsecureuisvc_jni.so \
     vendor/google/bonito/proprietary/product/lib64/vendor.google.wifi_ext@1.0.so:$(TARGET_COPY_OUT_PRODUCT)/lib64/vendor.google.wifi_ext@1.0.so \
     vendor/google/bonito/proprietary/product/lib64/vendor.qti.hardware.qdutils_disp@1.0.so:$(TARGET_COPY_OUT_PRODUCT)/lib64/vendor.qti.hardware.qdutils_disp@1.0.so \
-    vendor/google/bonito/proprietary/product/priv-app/EuiccSupportPixel/esim-full-v0.img:$(TARGET_COPY_OUT_PRODUCT)/priv-app/EuiccSupportPixel/esim-full-v0.img \
-    vendor/google/bonito/proprietary/product/priv-app/EuiccSupportPixel/esim-v1.img:$(TARGET_COPY_OUT_PRODUCT)/priv-app/EuiccSupportPixel/esim-v1.img \
     vendor/google/bonito/proprietary/product/priv-app/MyVerizonServices/lib/arm64/libakuaf.so:$(TARGET_COPY_OUT_PRODUCT)/priv-app/MyVerizonServices/lib/arm64/libakuaf.so \
     vendor/google/bonito/proprietary/product/priv-app/MyVerizonServices/lib/arm64/libmotricity.so:$(TARGET_COPY_OUT_PRODUCT)/priv-app/MyVerizonServices/lib/arm64/libmotricity.so \
     vendor/google/bonito/proprietary/product/etc/cne/andsfCne.xml:$(TARGET_COPY_OUT_PRODUCT)/etc/cne/andsfCne.xml \
@@ -189,8 +187,6 @@ PRODUCT_PACKAGES += \
     DCMO \
     DMService \
     DiagMon \
-    EuiccGoogle \
-    EuiccSupportPixel \
     HotwordEnrollmentOKGoogleRT5514 \
     HotwordEnrollmentXGoogleRT5514 \
     MyVerizonServices \
